cmake/globalmacros.cmake

```diff
@@ -203,6 +203,12 @@ function(ivw_private_setup_module_data)
     # In that case set to INVIWO<NAME>MODULE_description
     if(EXISTS "${${mod}_path}/readme.md")
         file(READ "${${mod}_path}/readme.md" description)
+        # truncate description since some readme files are quite substantial
+        string(LENGTH "${description}" desc_len)
+        if(desc_len GREATER 250)
+            string(SUBSTRING "${description}" 0 250 description)
+            string(JOIN "" description "${description}" "...")
+        endif()
```

Comment:
If the description is 251 characters, we will remove one and add 3 dots resulting in 253 characters ;)

Reply:
as if that matters :)